ur_robot_driver/src/ros/hardware_interface.cpp

```diff
@@ -65,6 +65,18 @@ bool HardwareInterface::init(ros::NodeHandle& root_nh, ros::NodeHandle& robot_hw
     return false;
   }
 
+  // Port that will be opened to communicate between the driver and the robot controller.
+  uint32_t reverse_port;
+  int reverse_port_tmp;
+  robot_hw_nh.getParam("reverse_port", reverse_port_tmp);
+  reverse_port = (uint32_t)reverse_port_tmp;
```

Comment:
We can simplify this to
```c++
int reverse_port = robot_hw_nh.param("reverse_port", 50001);
```
The casting will take place when passing it to the `URDriver` constructor